modules/aggregation-layers/src/heatmap-layer/heatmap-layer.ts

```diff
@@ -425,10 +425,12 @@ export default class HeatmapLayer<
   _createWeightsTransform(shaders = {}) {
     let {weightsTransform} = this.state;
     const {weightsTexture} = this.state;
+    // @ts-ignore
     weightsTransform?.delete();
```

Comment:
This file is very high on my list to get to after getting some more things done on the "Transform" class, so I'm not worried about the `@ts-ignore` usage here, but happy to suggest fixes for any of these if that's helpful at this stage.